docs/content/kitty-items/index.md

```diff
@@ -13,6 +13,10 @@ The following chapters use the project to explain the components and concepts of
 - add a new kind of NFT to the marketplace
 - interact with the Flow blockchain from within a web application
 
+## Video Walkthrough
```

Comment:
nit: Maybe describe in one sentence what this is and that it run through the same steps laid out on each page?

Reply:
Sounds good, I added an introduction outlining the format and explicitly stated that the videos are optional (depending on learning style)